app/commands/graph.js

```diff
@@ -0,0 +1,11 @@
+const prepareAnswer = require('../utils/prepareAnswer');
+
+function graph(body, bot, chatId, n = 10) {
```

Comment:
Since, we are passing `n`, how about supporting the N parameter? That would be cool, please.

Reply:
Sure. I will add it too.